resources/css/main.css

```diff
@@ -342,6 +352,160 @@ hr {
             transform: scale(1.4);
   }
 }
+.cardPopup {
+  position: fixed;
+  width: 90vw;
+  height: 95vh;
+  margin: 0 auto;
+  top: 3%;
+  left: 0;
+  right: 0;
+  background-color: #fff;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: start;
+      -ms-flex-align: start;
+          align-items: flex-start;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+      -ms-flex-direction: column;
+          flex-direction: column;
+  padding: 1.6rem;
+  border: 1px solid #dfe1e6;
+  border-radius: 1.6rem;
+}
+@media only screen and (min-width: 768px) {
+  .cardPopup {
+    display: grid;
+    grid-template-columns: 33.3% 25% auto;
+    grid-template-rows: 15% 50% 30%;
+    max-width: 70%;
+    gap: 2rem;
+  }
+  .cardPopup .cardtop {
+    grid-column: 1/span 2;
+  }
+  .cardPopup .cardimage {
```

Comment:
- [ ] Your image is stretching, kindly fix this so your popup is similar to the Figma design

|Your design|Figma design|
|-|-|
|![Screenshot_20220617_010613](https://user-images.githubusercontent.com/25227022/174197852-aa8e07c6-6777-4d33-9964-0860e31794bd.png)|![Screenshot_20220617_010635](https://user-images.githubusercontent.com/25227022/174197869-bd74568d-61de-405c-9a90-0b2cedae2bf1.png)|